examples/qaoa.py

```diff
@@ -1,155 +1,162 @@
 """Runs the Quantum Approximate Optimization Algorithm on Max-Cut.
 
-=== EXAMPLE OUTPUT ===
+=== EXAMPLE CIRCUIT ===
+  x0    x1    x2     x3
+  │     │     │      │
+  H     H     H      H
+  │     │     │      │
+  │     │     Y^-0.5 Y^-0.5
+  │     │     │      │
+  │     @─────@      │
+  │     │     │      │
+  │     │     Y^0.5  │
+  │     │     │      │
+  │     │     Rz(0)  │
```

Comment:
I'm just noticing these Rz(0), Rx(0) gates - we should filter these out I think at least in the example by passing the circuit through a `cirq.DropNegligible().optimize_circuit(circuit)`.